fighthealthinsurance/ml/ml_inference.py

```
@@ -0,0 +1,46 @@
+                model._infer_no_context(
+                    system_prompts=system_prompts,
+                    prompt=prompt,
+                    temperature=temperature,
+                ),
+                timeout=timeout,
+            )
+            if result and len(str(result).strip()) > min_length:
+                return str(result).strip()
+        except asyncio.TimeoutError:
+            logger.warning(f"Timeout on {label} with {model}")
+        except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)